deepchecks/core/serialization/suite_result/widget.py

```diff
@@ -150,16 +150,18 @@ def prepare_failures(
         if len(failures) == 0:
             children = (HTML(value='<p>No outputs to show.</p>'),)
         else:
-            df = create_failures_dataframe(failures)
-            table = DataFrameSerializer(df.style.hide_index()).serialize()
+            styler = create_failures_dataframe(failures).style
+            # style.hide_index() was deprecated in the latest versions and new method was added
+            styler = styler.hide(axis='index') if hasattr(styler, 'hide') else styler.hide_index()
```

Comment:
Do our requirements limit to versions that have this new method for sure?

Reply:
we require `pandas>=1.1.5`, but `hide()` method was added in version `pandas==1.4.0`
therefore I added a condition